src/pages/en/reference/api-reference.md

```diff
@@ -129,16 +129,33 @@ Astro.response.headers.set('Set-Cookie', 'a=b; Path=/;');
 
 ### `Astro.canonicalURL`
 
-The [canonical URL][canonical] of the current page. If the `site` option is set, the site's origin will be the origin of this URL.
+:::caution[Deprecated]
+Use [`Astro.url`](#astrourl) to construct your own canonical URL. 
+:::
+
+The [canonical URL][canonical] of the current page.
 
-You can also use the `canonicalURL` to grab the current page's `pathname`.
+### `Astro.url`
+
```

Comment:
In line with [what we did for `Astro.clientAddress`](https://docs.astro.build/en/reference/api-reference/#astroclientaddress) we can use the `<Since />` component here to highlight when this was added.

```suggestion

<Since v="1.0.0-rc" />

```

Reply:
Nice! I forgot about this!